dvc/output/base.py

```diff
@@ -21,15 +21,32 @@
 logger = logging.getLogger(__name__)
 
 
-class OutputDoesNotExistError(DvcException):
+def _format_msg(msg, path: "BaseOutput"):
```

Comment:
@efiop, would love to hear what you think. This one is a major pain here, if it's solved, we can start using `mypy` more.

Reply:
Though, now these exceptions won't be two different things. :(